imap_processing/lo/l1a/lo_cdf_attrs.py

```diff
@@ -58,3 +64,109 @@
     units="seconds",
     label_axis="ToF",
 )
+
+### Science Counts Attributes ###
+lo_anode_rate_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=GlobalConstants.INT_FILLVAL,
```

Comment:
The fill val is negative, so do you mean the validmin should change? I thought the fill vals didn't count as a valid number to account for in the range since it's meant to indicate missing data.

Reply:
Then this would read as `valid maximum is -199999` or whatever it is. I think this validmax should be the maximum value this rate could take which would be more like `99999` or a positive number. The max shouldn't be less than the min and I'm guessing SPDF would complain about.

Reply:
Oh sorry, I misunderstood your comment and didn't notice that in the code. Yeah, I put the wrong value there for validmax. I'll update that to INT_MAXVAL